src/bo4e/bo/angebot.py

```diff
@@ -0,0 +1,92 @@
+"""
+Contains Angebot class and corresponding marshmallow schema for de-/serialization
+"""
+
+from datetime import datetime
+from typing import List, Optional
+
+import attr
+from marshmallow import fields
+from marshmallow_enum import EnumField  # type:ignore[import]
+
+from bo4e.bo.ansprechpartner import Ansprechpartner, AnsprechpartnerSchema
+from bo4e.bo.geschaeftsobjekt import Geschaeftsobjekt, GeschaeftsobjektSchema
+from bo4e.bo.geschaeftspartner import Geschaeftspartner, GeschaeftspartnerSchema
+from bo4e.com.angebotsvariante import Angebotsvariante, AngebotsvarianteSchema
+from bo4e.enum.botyp import BoTyp
+from bo4e.enum.sparte import Sparte
+from bo4e.validators import check_list_length_at_least_one
+
+
+# pylint: disable=too-few-public-methods, too-many-instance-attributes
+@attr.s(auto_attribs=True, kw_only=True)
+class Angebot(Geschaeftsobjekt):
+    """
+    Mit diesem BO kann ein Versorgungsangebot zur Strom- oder Gasversorgung oder die Teilnahme an einer Ausschreibung
+    übertragen werden. Es können verschiedene Varianten enthalten sein (z.B. ein- und mehrjährige Laufzeit).
+    Innerhalb jeder Variante können Teile enthalten sein, die jeweils für eine oder mehrere Marktlokationen erstellt
+    werden.
+    """
+
+    bo_typ: BoTyp = attr.ib(default=BoTyp.ANGEBOT)
+    # required attributes
+    #: Eindeutige Nummer des Angebotes
+    angebotsnummer: str = attr.ib(validator=attr.validators.instance_of(str))
+    #: Erstellungsdatum des Angebots
+    angebotsdatum: datetime = attr.ib(validator=attr.validators.instance_of(datetime))
+    #: Sparte, für die das Angebot abgegeben wird (Strom/Gas)
+    sparte: Sparte = attr.ib(validator=attr.validators.instance_of(Sparte))
+    #: 	Link auf den Ersteller des Angebots
```

Comment:
```suggestion
    #: Ersteller des Angebots
```
ist ja kein link sondern einfach ein objekt